repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -211,7 +211,7 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
     val maybeForwardedHostUri = Try(maybeForwardedHost
       .map(forwardedHost => new URIBuilder(s"${request.getScheme}://$forwardedHost"))
       .map(uri => uri.setPort(normalizePort(uri.getPort, uri.getScheme)).setHost(normalizeHost(uri.getHost)).build()))
-    lazy val hostUri = new URIBuilder().setScheme(request.getScheme).setHost(normalizeHost(request.getServerName))
+    lazy val hostUri = new URIBuilder("").setScheme(request.getScheme).setHost(normalizeHost(request.getServerName))
```

Comment:
What did you bring in that forced this change? I know it was talked about at one point but i don't recall.

Reply:
jaeger-apachehttpclient (https://github.com/jaegertracing/jaeger-client-java/blob/v0.24.0/jaeger-apachehttpclient/build.gradle#L5-L7)

Reply:
Mainly this in 4.5.3 - https://github.com/apache/httpcomponents-client/blob/4.5.3/httpclient/src/main/java/org/apache/http/client/utils/URIBuilder.java#L493-L497 vs 4.4.1 - https://github.com/apache/httpcomponents-client/blob/4.4.1/httpclient/src/main/java/org/apache/http/client/utils/URIBuilder.java#L496-L499